packages/ember-routing/lib/system/router.js

```diff
@@ -585,7 +585,7 @@ const EmberRouter = EmberObject.extend(Evented, {
       seen[name] = true;
 
       if (!handler) {
-        let DefaultRoute = routeOwner._lookupFactory('route:basic');
+        let DefaultRoute = routeOwner.factoryFor('route:basic').class;
```

Comment:
So this doesn't seem to work for generated controllers due to the fact `.proto()` called right after we generate the controller. https://github.com/emberjs/ember.js/blob/192017b71070d5a32f24ec215ad75dd702d0c0bc/packages/ember-routing/lib/system/route.js#L193